vortex-array/src/arrays/bool/compute/uncompressed_size.rs

```diff
@@ -0,0 +1,11 @@
+use vortex_error::VortexResult;
+
+use crate::arrays::{BoolArray, BoolEncoding};
+use crate::compute::UncompressedSizeFn;
+use crate::nbytes::NBytes;
+
+impl UncompressedSizeFn<&BoolArray> for BoolEncoding {
+    fn uncompressed_size(&self, array: &BoolArray) -> VortexResult<usize> {
+        Ok(array.nbytes())
```

Comment:
I don't think this is true, since nbytes includes metadata. Don't we just want this to be the data buffers?

Reply:
Like we talked, I removed the metadata from `NBytes::nbytes` so this should be correct where its used.

Reply:
I don't think it's quite right, since you may have a compressed validity buffer.

I think you need `self.buffer().len() + self.validity().uncompressed_size()` or something